descwl_coadd/online_coadds.py

```diff
@@ -83,18 +81,18 @@ def make_online_stacker(stats_ctrl, coadd_bbox):
         The coadd bbox
     """
 
-    # dims is C/numpy ordering
-    dims = get_dims_from_bbox(coadd_bbox)
-
     mask_map = AssembleCoaddTask.setRejectedMaskMapping(stats_ctrl)
 
+    cefiv = stats_ctrl.getCalcErrorFromInputVariance()
+
+    mask_threshold_dict = stats_ctrl_to_threshold_dict(stats_ctrl)
```

Comment:
@erykoff I added this, which I suppose is just using defaults

For EDGE at least we don't want *any* to make it through, so how do I make that happen?  Sorry for so many detailed questions.